From a648336c0e837c016bba5f19ec683d09f91471e5 Mon Sep 17 00:00:00 2001
From: Stefan Hajnoczi <stefanha@redhat.com>
Date: Tue, 7 Oct 2014 14:06:57 +0200
Subject: [PATCH 39/43] trace: install trace-events file

Message-id: <1412690820-31016-9-git-send-email-stefanha@redhat.com>
Patchwork-id: 61613
O-Subject: [RHEL7.1 qemu-kvm PATCH 08/11] trace: install trace-events file
Bugzilla: 1088112
RH-Acked-by: Laszlo Ersek <lersek@redhat.com>
RH-Acked-by: Miroslav Rezanina <mrezanin@redhat.com>
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>

Install the ./trace-events file into the data directory.  This file
contains the list of trace events that were built into QEMU at
compile-time.

The file is a handy reference for the set of trace events that the QEMU
binary was built with.  It is also needed by the simpletrace.py tool
that parses binary trace data either emitted from QEMU when built with
--enable-trace-backend=simple or by the SystemTap simpletrace script
that QEMU provides.

Signed-off-by: Stefan Hajnoczi <stefanha@redhat.com>
Message-id: 1411486175-3017-1-git-send-email-stefanha@redhat.com
(cherry picked from commit 89ae5831a53e2d30a370e9a30fdb35da5a8f89aa)
Signed-off-by: Stefan Hajnoczi <stefanha@redhat.com>

Downstream needs to install trace-events in qemu-kvm.spec.template to
prevent rpm check "unpackaged file" errors.

Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 Makefile                      | 1 +
 redhat/qemu-kvm.spec.template | 3 ++-
 2 files changed, 3 insertions(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index f504754..1b0b4f7 100644
--- a/Makefile
+++ b/Makefile
@@ -406,6 +406,7 @@ endif
 	set -e; for x in $(KEYMAPS); do \
 		$(INSTALL_DATA) $(SRC_PATH)/pc-bios/keymaps/$$x "$(DESTDIR)$(qemu_datadir)/keymaps"; \
 	done
+	$(INSTALL_DATA) $(SRC_PATH)/trace-events "$(DESTDIR)$(qemu_datadir)/trace-events"
 	for d in $(TARGET_DIRS); do \
 	$(MAKE) -C $$d $@ || exit 1 ; \
         done
-- 
1.8.3.1

